samples/file-gateway/resources/bills/test.ignore

```diff
@@ -0,0 +1 @@
+Dummy file created to add 'bills' directory into git
```

Comment:
Empty file would be okay

Reply:
I think without the description it might not be obvious to some people why its there :)